src/dep.cc

```diff
@@ -918,7 +918,12 @@ class DepBuilder {
   }
 
   Evaluator* ev_;
-  map<Symbol, RuleMerger> rules_;
+  struct TargetComp {
```

Comment:
I was about to be smart and try to come up with a way to re-use `std:less` but then I realized that it's best not to dabble in the dark arts.